### PR TITLE
[2.8.x] Upgrade Scalafmt to latest release of 2.4.x serias (backport #11524)

### PR DESCRIPTION
**Pull request by ihostage**
This is a manual backport of pull request #11524

### CODE REVIEW

File: core/play-guice/src/test/scala/play/api/inject/guice/GuiceInjectorBuilderSpec.scala [hunk 1]
```diff
@@ -188,7 +188,10 @@ class GuiceInjectorBuilderSpec extends Specification {
         )
         .disable[GuiceInjectorBuilderSpec.EnvironmentModule]
         .disable[GuiceInjectorBuilderSpec.JavaEnvironmentModule]
-        .disable(classOf[GuiceInjectorBuilderSpec.AModule], classOf[GuiceInjectorBuilderSpec.CModule]) // C won't be disabled
+        .disable(
+          classOf[GuiceInjectorBuilderSpec.AModule],
+          classOf[GuiceInjectorBuilderSpec.CModule]
+        ) // C won't be disabled
```
[the remark this answers is not in the document]
Comment (ihostage) [reply core/play-guice/src/test/scala/play/api/inject/guice/GuiceInjectorBuilderSpec.scala:194]: 😱 My fault 😞

File: core/play-integration-test/src/it/scala/play/it/http/JavaResultsHandlingSpec.scala [hunk 2]
```diff
@@ -854,7 +832,13 @@ trait JavaResultsHandlingSpec
...
           val objectNode = Json.newObject
           objectNode.put("foo", "bar")
-          Results.ok().sendJson(objectNode, false, Optional.of("file.txt")) // even though the extension is txt, the content-type is json
+          Results
+            .ok()
+            .sendJson(
+              objectNode,
+              false,
+              Optional.of("file.txt")
+            ) // even though the extension is txt, the content-type is json
```
Comment (ihostage) [inline core/play-integration-test/src/it/scala/play/it/http/JavaResultsHandlingSpec.scala:841]: plus one

File: core/play-integration-test/src/it/scala/play/it/http/ScalaResultsSpec.scala [hunk 3]
```diff
@@ -56,7 +56,9 @@ class ScalaResultsSpec extends PlaySpecification {
 
   "bake cookies should not depends on global state" in withApplication("play.allowGlobalApplication" -> false) {
     implicit app =>
-      Ok.bakeCookies(cookieHeaderEncoding, sessionBaker, flashBaker) must not(beNull) // we are interested just that it executes without global state
+      Ok.bakeCookies(cookieHeaderEncoding, sessionBaker, flashBaker) must not(
+        beNull
+      ) // we are interested just that it executes without global state
```
Comment (ihostage) [inline core/play-integration-test/src/it/scala/play/it/http/ScalaResultsSpec.scala:61]: plus one

File: web/play-filters-helpers/src/main/scala/play/filters/csrf/CSRFActions.scala [hunk 4]
```diff
@@ -179,7 +179,9 @@ class CSRFAction(
           }
         }))
         .splitWhen(_ => false)
-        .prefixAndTail(0) // TODO rewrite BodyHandler such that it emits sub-source then we can avoid all these dancing around
+        .prefixAndTail(
+          0
+        ) // TODO rewrite BodyHandler such that it emits sub-source then we can avoid all these dancing around
```
Comment (ihostage) [inline web/play-filters-helpers/src/main/scala/play/filters/csrf/CSRFActions.scala:184]: catched

File: web/play-openid/src/main/scala/play/api/libs/openid/OpenIdClient.scala [hunk 5]
```diff
@@ -314,7 +311,11 @@ private[openid] object Discovery {
           .findFirstIn(response.body)
           .orElse(delegateRegex.findFirstIn(response.body))
           .flatMap(extractHref(_))
-        OpenIDServer("http://specs.openid.net/auth/2.0/signon", url, delegate) //protocol version due to http://openid.net/specs/openid-authentication-2_0.html#html_disco
+        OpenIDServer(
+          "http://specs.openid.net/auth/2.0/signon",
+          url,
+          delegate
+        ) //protocol version due to http://openid.net/specs/openid-authentication-2_0.html#html_disco
```
Comment (ihostage) [inline web/play-openid/src/main/scala/play/api/libs/openid/OpenIdClient.scala:318]: catched

**Review by ihostage — COMMENTED** [review]
Plus several 🤦‍♂️

File: core/play-guice/src/test/scala/play/api/inject/guice/GuiceInjectorBuilderSpec.scala (hunk 1 above)
[the remark this answers is not in the document]
Comment (mkurz) [reply core/play-guice/src/test/scala/play/api/inject/guice/GuiceInjectorBuilderSpec.scala]: Thanks! I didn't really go through the diffs anymore, I know you fixed all :wink:

**Pull request merged**